examples/api_test.py

```diff
@@ -129,6 +129,19 @@
 # A few basic checks of the current state
 assert(len(ips) > 0)
 
+# Getting config items under indexed key
+print("Accessing indexed config entries")
+count = 0
+networks = len(set(container.get_config_item('lxc.network')))
+network_keys = ['lxc.network.type','lxc.network.link','lxc.network.flags','lxc.netwo
+rk.hwaddr',]
+while (count < networks):
+    for nk in network_keys:
+        new_key = nk.replace('lxc.network.','lxc.network.{}.'.format(count))        
+        print(container.get_config_item(new_key))
+    count +=1
+    
```

Comment:
Wouldn't this be cleaner that way (untested):

```python
for idx, interface in enumerate(c.get_config_item("lxc.net")):
    for attribute in ("type", "link", "flags", "hwaddr"):
        key = "lxc.net.%s.%s" % (idx, attribute)
        print("%s = %s" % (key, c.get_config_item(key)))
```

Note the use of `lxc.net` rather than `lxc.network` as the code should be aimed at liblxc 3.0 and higher.

Reply:
much more cleaner :) thanks

Reply:
checked. Working ok with lxc.network. Looks like lxc.net not yet available in python3 bindinds.

Reply:
What liblxc version are you running? `lxc.net` needs 2.1 or higher, I confirmed that `lxc.network` doesn't work here but `lxc.net` does (on 3.0.2).

Reply:
Debian 9 stretch stable, python3-lxc-3.0.2 latest release
lxc package 2.0.7-2+deb9u2
I checked for you and an updated version of lxc is not available stretch-backports
Debian buster and sid have lxc 2.0.9

Reply:
That's an odd mix, you're certainly not meant to use python3-lxc 3.0.x with lxc 2.0.x :)

Anyway, can you update your pull request to match the suggested version (with `lxc.net`)?
Then I think we can merge it.